Assets/Scripts/Interactables/Dialogue/Conversation.cs

```diff
@@ -5,17 +5,17 @@
 [System.Serializable]
 public struct Line
 {
-    public bool isBirdie;
+    public string id;
 
     [TextArea(2, 5)]

```

Comment:
Can we make it use this instead :)
https://github.com/dbrizov/NaughtyAttributes#resizabletextarea

(just put `using NaughtyAttributes;` at the top and you can use it)